project/scaffold.sbt

```diff
@@ -0,0 +1,5 @@
+// TODO: do we need this?
```

Comment:
copyright?

Reply:
there is no copyright in the build.properties files for other services

Reply:
This file is an .sbt file, other .sbt files have it. We don't need for this one?